artifactory/manager.go

```diff
@@ -83,14 +83,14 @@ func (sm *ArtifactoryServicesManager) ReadRemoteFile(readPath string) (io.ReadCl
 	return readFileService.ReadRemoteFile(readPath)
 }
 
-func (sm *ArtifactoryServicesManager) DownloadFiles(params services.DownloadParams) ([]utils.FileInfo, int, error) {
+func (sm *ArtifactoryServicesManager) DownloadFiles(params ...services.DownloadParams) ([]utils.FileInfo, int, error) {
```

Comment:
The README should be updated with the new API signature.

Reply:
The README is not affected by this change.